src/tsconfig.spec.json

```diff
@@ -2,20 +2,10 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "outDir": "../out-tsc/spec",
-    "module": "commonjs",
-    "target": "es5",
     "baseUrl": "",
-    "types": [
-      "jasmine",
-      "node"
-    ]
+    "target": "es5",
```

Comment:
`target: es5` is a fix for karma/phantom test runner failing with `Use of reserved word 'class'`